files/ru/web/accessibility/aria/roles/checkbox_role/index.md

```diff
@@ -26,7 +26,7 @@ translation_of: Web/Accessibility/ARIA/Roles/checkbox_role
 
 Разработчику необходимо динамически изменять значение атрибута `aria-checked` при активации чекбокса.
 
-### Связанные WAI-ARIA Роли, Состояния, и Свойства
+### Связанные ARIA-роли, ARIA-состояния и ARIA-свойства
 
 <!---->
```

Comment:
Could you please remove these artifacts too?

Reply:
fixed